### ISSUE TITLE
how to switch to CK+ in visualize.py

### ISSUE THREAD

**Issue by Lee-N** [WuJie1010/Facial-Expression-Recognition.Pytorch#72]
I trained a CK+_VGG19 model by 
python mainpro_CK+.py --model VGG19 --bs 128 --lr 0.01 --fold 1
and the confusion matrix shows that the accuracy is 93.939%.
Then I changed visualize.py 
checkpoint=torch.load(os.path.join('FER2013_VGG19','PrivateTest_model.t7'),map_location=torch.device('cpu'))
net.load_state_dict(checkpoint['net'])
to 
checkpoint=torch.load(os.path.join('CK+_VGG19','Test_model.t7'),map_location=torch.device('cpu'))
net.load_state_dict(checkpoint['net'].state_dict())
Then I put my own image into images fold to test
It run successfuly
but the result is quite inaccurate 
I dont konw where the problem is
Please help me....

**Comment by WuJie1010** [comment]
please train ('CK+_VGG19','Test_model.t7') by yourself, and put in this dir.
